freqtrade/constants.py

```diff
@@ -266,6 +266,6 @@
         'stake_amount',
         'dry_run',
         'bid_strategy',
-        'telegram'
+        'unfilledtimeout',
```

Comment:
This section is missed in configuration.md, btw...

Reply:
no - line 58 and 59 (i had to check myself when i changed this) :D